packages/samples/atomic-react/src/pages/FoldedResultListPage.tsx

```diff
@@ -58,7 +60,10 @@ function MyTemplate(result: FoldedResult) {
         )}
       </AtomicResultSectionBadges>
       <AtomicResultSectionVisual>
-        <img loading="lazy" src="https://picsum.photos/350" />
+        <AtomicResultImage
+          field="ec_images"
+          fallback="https://picsum.photos/350"
```

Comment:
Guess `ec_images` doesn't make sense for folded results. But this could demonstrate usage of `fallback`?

Reply:
Sure, good idea!